File: app/src/main/java/com/example/tsj/service/model/RequestModel.java
package com.example.tsj.service.model;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class RequestModel {
    @SerializedName("Address")
    @Expose
    private String address;
    @SerializedName("Entrance")
    @Expose
    private Integer entrance;
    @SerializedName("Floor")
    @Expose
    private Integer floor;
    @SerializedName("Id")
    @Expose
    private Integer id;
    @SerializedName("RequestTypeName")
    @Expose
    private String requestTypeName;
    @SerializedName("IsEditableAndCloseable")
    @Expose
    private Boolean isEditableAndCloseable;
    @SerializedName("Description")
    @Expose
    private String description;

    public String getAddress() {
        return address;
    }

    public void setAddress(String address) {
        this.address = address;
    }

    public Integer getEntrance() {
        return entrance;
    }

    public void setEntrance(Integer entrance) {
        this.entrance = entrance;
    }

    public Integer getFloor() {
        return floor;
    }

    public void setFloor(Integer floor) {
        this.floor = floor;
    }

    public Integer getId() {
        return id;
    }

    public void setId(Integer id) {
        this.id = id;
    }

    public String getRequestTypeName() {
        return requestTypeName;
    }

    public void setRequestTypeName(String requestTypeName) {
        this.requestTypeName = requestTypeName;
    }

    public Boolean getEditableAndCloseable() {
        return isEditableAndCloseable;
    }

    public void setEditableAndCloseable(Boolean editableAndCloseable) {
        isEditableAndCloseable = editableAndCloseable;
    }

    public String getDescription() {
        return description;
    }

    public void setDescription(String description) {
        this.description = description;
    }
}

File: app/src/main/java/com/timelysoft/tsjdomcom/service/model/RequestModel.kt
package com.timelysoft.tsjdomcom.service.model

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class RequestModel {
    @SerializedName("Address")
    @Expose
    var address: String = ""
    @SerializedName("Entrance")
    @Expose
    var entrance: Int = 0
    @SerializedName("Floor")
    @Expose
    var floor: Int = 0

    @SerializedName("CreatedDate")
    @Expose
    var createdDate: String = ""

    @SerializedName("Id")
    @Expose
    var id: Int = 0
    @SerializedName("RequestTypeName")
    @Expose
    var requestTypeName: String = ""

    @SerializedName("StatusName")
    @Expose
    var statusName: String = ""

    @SerializedName("StatusDate")
    @Expose
    var statusDate: String = ""

    @SerializedName("IsEditableAndCloseable")
    @Expose
    var editableAndCloseable: Boolean = false
    @SerializedName("Description")
    @Expose
    var description: String = ""

}